crowbar_framework/app/models/ipmi_service.rb

```diff
@@ -62,16 +60,15 @@ def transition(inst, name, state)
     #
     if state == "discovered"
       @logger.debug("IPMI transition: discovered state for #{name}")
-      db = Proposal.where(barclamp: "ipmi", name: inst).first
-      role = RoleObject.find_role_by_name "ipmi-config-#{inst}"
-      result = add_role_to_instance_and_node("ipmi", inst, name, db, role, "ipmi-configure")
+      result = add_role_to_proposal_instance_and_node("ipmi", inst, name, "ipmi-configure")
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `result`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)